demo_package_exporter/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use rmf_site_format::Workcell;
+use package_exporter::{generate_package,PackageContext, Person};
+
+fn main() {
+    let args: Vec<String> = std::env::args().collect();
+    if args.len() != 3 {
+        eprintln!("Usage: {} <workcell_file> <output_directory>", args[0]);
+    }
+    let workcell_file_path = &args[1];
+    let output_directory = &args[2];
+
+    let package_context = PackageContext {
+        project_name: "output_package".to_string(),
```

Comment:
This hardcodes the package name to `output_package` while the rest of the executable uses `test_package`. As a consequence paths look like `package://test_package/[...]` which cannot be resolved.

```suggestion
        project_name: "test_package".to_string(),
```

Reply:
Should be fixed in c822bd5.